src/llama_stack_api/batches/__init__.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
not a fan of keeping this in the init.py -- why can't we move it to `batches/api.py` or `batches/batches.py`? 

secondly, why do we want to split `fastapi_routes.py` (i.e., the annotations) from the API signature? I guess it is cleaner but now there is boilerplate and we need to keep these two things in sync!